app/src/main/kotlin/com/wire/android/migration/userDatabase/ScalaMessageDAO.kt

```diff
@@ -101,13 +102,13 @@ class ScalaMessageDAO(
             accumulator
         }
 
-    private fun messagesFromConversation(scalaConversation: ScalaConversationData): List<ScalaMessageData> {
+    private fun messagesFromConversation(scalaConversation: Conversation): List<ScalaMessageData> {
         val cursor = db.rawQuery(
             "SELECT *" + // Assets are required, otherwise we get exception "requesting column name with table name".
                     "FROM $MESSAGES_TABLE_NAME AS m " +
                     "LEFT JOIN $ASSETS_TABLE_NAME ON m.$COLUMN_ASSET_ID = $ASSETS_TABLE_NAME.$COLUMN_ID " +
                     "WHERE m.$COLUMN_CONVERSATION_ID = ?" +
-                    "ORDER BY m.$COLUMN_TIME ASC ", arrayOf(scalaConversation.id)
+                    "ORDER BY m.$COLUMN_TIME ASC ", arrayOf(scalaConversation.id.value)
```

Comment:
Holly.

Was this working before with `scalaConversation.id`? 🤔

Reply:
Previously it was `ScalaConversationData` and now it uses `Conversation`, that's why it worked before 😄